### DISCUSSION TITLE
Feature Request: Git Pull

### DISCUSSION THREAD

**Discussion by rkho** [standardnotes/forum#2486] [Suggestions]
I love being able to push files (and override them!) to a Github repository of mine, but is it possible to have some sort of synced notebook that lives in a folder/on my GH account?

The use case I have is that I've been using Github to store my own personal wikis/notes lately and this is the closest option for a well-designed text editor which lets me begin notes in iOS and then later sync them to GH without having to paste it into a text editor

**Comment by atmoio** [comment]
Would definitely be cool. Will probably leave this for community development, as we simply can't dedicate full time to something this niche.
